policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -98,6 +98,13 @@
           ci/bin/unlock-agent.sh
           {{ if has "plugin-compiler-fix-vendor" .Branchvals.Features -}}
           mkdir -p /go/src
+          {{/* 
```

Comment:
Rather than this being a template comment, this comment should be rendered into the templates so that it is clear.